easybuild/easyconfigs/c/ctffind/ctffind-4.1.14-foss-2021a-CUDA-11.3.1.eb

```
@@ -0,0 +1,47 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+#
+# Author: Ake Sandgren, HPC2N, Umea University
+
+easyblock = 'ConfigureMake'
+
+name = 'ctffind'
+version = '4.1.14'
+local_cudaversion = '11.3.1'
+versionsuffix = '-CUDA-%s' % local_cudaversion
+
+homepage = 'https://grigoriefflab.umassmed.edu/ctffind4'
+description = """Program for finding CTFs of electron micrographs."""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+toolchainopts = {'openmp': True}
+
+source_urls = ['https://grigoriefflab.umassmed.edu/sites/default/files/']
+sources = [SOURCELOWER_TAR_GZ]
+patches = ['%(name)s-%(version)s_asm-fix.patch']
+checksums = [
+    'db17b2ebeb3c3b2b3764e42b820cd50d19ccccf6956c64257bfe5d5ba6b40cb5',  # ctffind-4.1.14.tar.gz
+    'e6d468b3f1569e2d42e077573529dbc3035a03715c436d2349ccaaab63b64f28',  # ctffind-4.1.14_asm-fix.patch
+]
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('libjpeg-turbo', '2.0.6'),
+    ('LibTIFF', '4.2.0'),
+    ('GSL', '2.7'),
+    ('wxWidgets', '3.1.5'),
+    ('CUDAcore', local_cudaversion, '', True),
```

```suggestion
    ('CUDAcore', '11.3.1', '', True),
```
together with the other suggestion will probably make it easier to things like try-update-deps and such to work, so I think it would be beter to use that approach